Formula/cctz.rb

```diff
@@ -0,0 +1,62 @@
+class Cctz < Formula
+  desc "C++ library for translating between absolute and civil times"
+  homepage "https://github.com/google/cctz"
+  url "https://github.com/google/cctz/archive/v2.1.tar.gz"
+  sha256 "a86b9b2d339f5638a03a4bbcc1d8c632d2c429aecc44f5c548839c239c3b6e38"
+
```

Comment:
maybe add a `head` with `head "https://github.com/google/cctz.git"`?

Reply:
let's not